modules/core/mwaa/stack.py

```diff
@@ -180,6 +186,13 @@ def __init__(
                     effect=aws_iam.Effect.ALLOW,
                     resources=[f"arn:aws:iam::{account}:role/addf-*"],
                 ),
+                aws_iam.PolicyStatement(
```

Comment:
Most likely the `script/fix.sh` will need to rerun on this module to format

Reply:
I ran the fix and validate scripts on the mwaa module, looks good from my side